addons/web/static/src/legacy/js/core/smooth_scroll_on_drag.js

```diff
@@ -391,7 +391,15 @@ const SmoothScrollOnDrag = Class.extend(mixins.ParentedMixin, {
      */
     _onSmoothDragStart(ev, ui, onDragStartCallBack) {
```

Comment:
Behavior:
Note that it doesn't scroll on mass mailing (again, not sure if it should)

Reply:
> Behavior: Note that it doesn't scroll on mass mailing (again, not sure if it should)

I will check if it is fixable in a potential new PR

Reply:
@sobo-odoo I created a new task (3877644) for it 😬